src/components/MovieDetail/MovieDetail.js

```diff
@@ -2,22 +2,29 @@ import React from 'react';
 import './MovieDetail.css';
 
 const MovieDetail = ({movie}) => {
+
+  const formatDate = (movieReleaseDate) => {
+    const date = new Date(movieReleaseDate);
+    return date.toDateString();
+  };
+
+  const genres = movie.genres.join(' ');
+
   return (
-    <section>
-      <div className='detail-page-header'>
-        <h2>{movie.title}</h2>
-        <p>Genres: {movie.genres}</p>
-      </div>
-      <div className='movie-info'>
-        <img className='poster-image border' src={movie.poster_path} />
-        <article className='border'>
+    <section className='movie-detail-page'>
+      <div className='backdrop-img' style={{backgroundImage:`url(${movie.backdrop_path})`}}>
+        <article className='movie-spec movie-container border'>
+          <h2 className='movie-title'>{movie.title}</h2>
+          <p>Release Date: {formatDate(movie.release_date)}</p>
+          <p>Genres: {genres}</p>
+          <p>Runtime: {movie.runtime} minutes</p>
+          <p>Budget: ${new Intl.NumberFormat().format(movie.budget)}</p>
+          <p>Revenue: ${new Intl.NumberFormat().format(movie.revenue)}</p>
+        </article>
+        <article className='movie-info movie-container border'>
```

Comment:
I'm curious about how this will look when we're missing data. We might want to revisit this with more conditional rendering later.